.travis.yml

```diff
@@ -0,0 +1,48 @@
+sudo: false
+dist: trusty
+language: c
+addons:
+  apt:
+    packages:
+     - libfftw3-dev
+     - libgconf2-dev
+     - libgtk-3-dev
+     - libgtk-3-common
+     - libgtk-3-0
+     - libjpeg-dev
+     - libtiff4-dev
+     - liblcms2-dev
+     - libcurl4-gnutls-dev
+     - libexiv2-dev
+     - libflickcurl-dev
+     - libsqlite3-dev
+#     - libgphoto2-dev
+#     - libgphoto2-dev-doc
+     - libgphoto2-2-dev
+     - autoconf
+     - automake
+     - liblensfun-dev
+#     - libosmgpsmap-dev
+     - libsoup2.4-dev
+
+before_install:
+#  - apt-get update -qq
+#  - sudo apt-get install -y libfftw3-dev libgconf2-dev libgtk-3-dev libjpeg-dev libtiff4-dev liblcms2-dev libcurl4-gnutls-dev libexiv2-dev libflickcurl-dev libsqlite3-dev libgphoto2-2-dev autoconf automake liblensfun-dev liblensfun-data libsoup2.4-dev
```

Comment:
```suggestion
```

This stuff goes in package keys above anyway, can we just delete unused lines instead of cluttering the comments?